### PR TITLE
DS: Build Divider

### PR DESCRIPTION
**Pull request by nearbycoder**
Adds the basic divider component and storybook examples.

### CODE REVIEW

File: lib/components/divider/Divider.jsx [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import { Divider as ChakraDivider } from '@chakra-ui/react';
+
+export function Divider({ ...props }) {
```
Comment (caseymhunt) [inline lib/components/divider/Divider.jsx:4]: I think I've seen us using `...rest` and `...props` interchangeably. Any thoughts on some standardization / rules about which/when to use?

**Review by caseymhunt — APPROVED** [review]
👍🏼 lgtm

File: lib/components/divider/Divider.jsx (hunk 1 above)
Comment (nearbycoder) [reply lib/components/divider/Divider.jsx]: Updated, my thoughts around this was `...rest` indicates when you are splitting out some of the props vs. `...props` indicates all of them. Went ahead and updated it for consistency though.

Comment (caseymhunt) [reply lib/components/divider/Divider.jsx]: Oh yeah I can get down with that logic!

**Pull request merged**